code/modules/mob/living/blood.dm

```diff
@@ -194,6 +211,17 @@
 		return 0
 	for(var/obj/item/bodypart/bodypart as anything in bodyparts)
 		bleed_rate += bodypart.get_bleed_rate()
+
+	// Blood pressure system: less blood = lower blood pressure = slower bleeding
+	// Uses exponential curve for more dramatic slowdown near death
+	if(bleed_rate > 0 && blood_volume < BLOOD_VOLUME_NORMAL)
+		var/blood_pressure = blood_volume / BLOOD_VOLUME_NORMAL
+		// Square the pressure ratio for exponential slowdown
+		// At 50% blood: 0.5^2 = 25% bleed rate
+		// At 25% blood: 0.25^2 = 6.25% bleed rate
+		// At 10% blood: 0.1^2 = 1% bleed rate
+		bleed_rate *= (blood_pressure ** 2)
```

Comment:
food for thought: since this is so much dramatically slower to hit 0 blood volume now, we can possibly consider **massively** increasing the amount of oxyloss damage dealt at this point to expedite things along, or even call death() directly?

Reply:
I thought about making it so that reaching 0 blood volume just makes you die, but wasn't sure. That is probably fine, I think, but since killing strokes are now possible without having to behead somebody, I wanted to see how it would play it first.

Reply:
Decided to implement this after further testing. Update incoming.